### PR TITLE
Adapt to build for scarthgap

### PR DESCRIPTION
**Pull request by MykolaSuperman**

### CODE REVIEW

**Review by mykola-kobets-epam — APPROVED** [review]

File: policy/modules/services/dbus.te [hunk 1]
```diff
@@ -190,6 +190,10 @@ write_logging_runtime_dirs(system_dbusd_t)
 
 files_map_etc_files(system_dbusd_t)
 
+init_rw_script_stream_sockets(system_dbusd_t)
+
```
Comment (al1img) [inline policy/modules/services/dbus.te]: remove empty line

File: policy/modules/services/rpc.te [hunk 2]
```diff
@@ -363,6 +367,10 @@ fs_manage_tmpfs_dirs(nfsd_t)
 
 fs_list_hugetlbfs(nfsd_t)
 
+init_rw_script_stream_sockets(nfsd_t)
+
```
Comment (al1img) [inline policy/modules/services/rpc.te]: ditto

File: policy/modules/system/lvm.te [hunk 3]
```diff
@@ -229,6 +229,10 @@ fs_manage_bpf_dirs(lvm_t)
 init_search_keys(lvm_t)
 miscfiles_manage_generic_cert_files(lvm_t)
 
+init_rw_script_stream_sockets(lvm_t)
+
```
Comment (al1img) [inline policy/modules/system/lvm.te]: ditto

File: policy/modules/system/systemd.te [hunk 4]
```diff
@@ -960,6 +964,10 @@ mls_file_write_to_clearance(systemd_logind_t)
 
 files_map_etc_files(systemd_logind_t)
 
+init_rw_script_stream_sockets(systemd_logind_t)
+
```
Comment (al1img) [inline policy/modules/system/systemd.te]: ditto

File: policy/modules/system/systemd.te [hunk 5]
```diff
@@ -1565,6 +1581,11 @@ systemd_log_parse_environment(systemd_resolved_t)
 systemd_read_networkd_runtime(systemd_resolved_t)
 
 files_map_etc_files(systemd_resolved_t)
+init_rw_script_stream_sockets(systemd_resolved_t)
+
```
Comment (al1img) [inline policy/modules/system/systemd.te]: ditto

File: policy/modules/system/systemd.te (hunk 4 above)
Comment (MykolaSuperman) [reply policy/modules/system/systemd.te]: Done

File: policy/modules/system/systemd.te (hunk 5 above)
Comment (MykolaSuperman) [reply policy/modules/system/systemd.te]: Done

File: policy/modules/system/lvm.te (hunk 3 above)
Comment (MykolaSuperman) [reply policy/modules/system/lvm.te]: Done

File: policy/modules/services/rpc.te (hunk 2 above)
Comment (MykolaSuperman) [reply policy/modules/services/rpc.te]: Done

File: policy/modules/services/dbus.te (hunk 1 above)
Comment (MykolaSuperman) [reply policy/modules/services/dbus.te]: Done

**Review by al1img — APPROVED** [review]

**Review by mlohvynenko — APPROVED** [review]

**Pull request merged**